jac-uk/admin
# #1929 Timeline - QT Dates

## What's included?
Closes #1929 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Go to the timeline of an exercise and check if QT dates are mandatory.
- Critical analysis qualifying test (QT)
- Scenario test qualifying test (QT)
- Situational judgement qualifying test (QT)

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

https://user-images.githubusercontent.com/79906532/231148557-5472bb0a-3635-401c-929d-5fd48048b092.mov

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
